src/main/java/Main.java

```diff
@@ -53,10 +57,14 @@ public static void main(String[] args) {
         ViewManagerModel viewManagerModel = new ViewManagerModel();
         ViewManager viewManager = new ViewManager(views, cardLayout, viewManagerModel);
 
-        // Create User Panel
+        // Manage Users
         CreateUserPanelViewModel createUserPanelViewModel = new CreateUserPanelViewModel();
+        EditProfileViewModel editProfileViewModel = new EditProfileViewModel();
+        ManageUsersController manageUsersUseCaseFactory = ManageUsersUseCaseFactory.create(createUserPanelViewModel, editProfileViewModel);
```

Comment:
I think this variable could be changed to another name to make it more clear what it is (like manageUsersController).